[Blog] Why your growth team isn't winning (and how to fix it)

Lior539 opened the pull request:
Based off an interview I did with Alexey

Lior539 commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
```
Alexey are there any achievements or numbers I can add here that can help build up your credibility and authority on the content?

AlexeyMK replied:
Sure - I think alexeymk.com overall is a pretty good place to link, especially for now.

MasterClass - I grew the Growth Eng org from 8 to ~25, and the valuation quadrupled during my 3 years there. That feel about right?

AlexeyMK replied:
let's incldue here that these days Alexey is a [Growth Engineering Advisor](alexeymk.com/growth-eng)

Lior539 replied:
> MasterClass - I grew the Growth Eng org from 8 to ~25, and the valuation quadrupled during my 3 years there. That feel about right?

@AlexeyMK  Hmm any other metrics/achievements we could include? I think stating the valuation growth is not an ideal, since we cannot attribute it to you directly.

AlexeyMK replied:
heh, those are the public ones.  MasterClass is pretty tight-lipped about internal ones.

>   we cannot attribute it to you directly.

not inviting you to my next performance review, lol

Lior539 replied:
Will rework to focus on your experience in growth. I want to avoid valuations, as 2021 had crazy valuations and people may be cynical

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+## Problem 1: Your team is moving too slowly
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
```
I think we're basically paraphrasing this piece here https://alexeymk.com/2021/04/01/a-toolset-for-tackling-technical-debt.html

Lior539 replied:
Thanks! Will link to it below

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of code quality being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```
The term here to use is Architecture Astronauts, I think: https://www.joelonsoftware.com/2001/04/21/dont-let-architecture-astronauts-scare-you/

Also
> slow velocity is usually a result of code quality being too high.
no, high code quality in the context of growth means the code is easy to iterate on.

The cause of slow velocity is over-engineering.

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+You keep trying new ideas, but none of your experiments are working. 
+
+"You need better ideas," says Alexey, "and, counter-intuitively, the way to do this is to spend more time adding clear estimates to your current ideas."
```
have a piece coming out end of month that will be good for this. Not public yet though: :https://paper.dropbox.com/doc/Newly-minted-growth-teams-have-about-6-months-to-demonstrate-impact.-Despite-a-wealth-of-ideas-theres-only-time-for-a-handful--B_pDhdC_5ixmVFOvDfv3t_p1Ag-PZay0adGqCHSSK9tvAttz

Lior539 replied:
Cool! We can link to it when it's ready

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause major incidents! And their goal must not be to avoid them.
```
"major incidents" is a bit too much.  I would say to cause _a few_ incidents.

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.
+- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
+
+Notice that none of the solutions involve writing tests, which are a **waste of time** for experiments that may only last for a week.
```
First, I think this is the key quote: https://twitter.com/rauchg/status/807626710350839808

Automated tests "are a waste of time" is probably too aggressive.   They're expensive overhead - valuable, but not good ROI so long as you are (a) executing tests manually and (b) have enough other safeguards in place.

Even still, every company has a unique set of criteria and needs to figure out whether automated tests for individual experiments make sense for them.  All I would add here is "consider what it would take to not write tests for individual short-lived experiments"

Lior539 replied:
changed to "can be a waste of time" and added "Instead, consider what it would take to comfortably ship changes without tests."

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+It's okay if your engineers are tired. Growth engineering may not be as fulfilling for some people, especially if they yearn to build deep and complex systems.
+
+It's common for engineers to start feeling bored after a few quarters, and there's no shame in it. A symptom of this is diminishing returns from the team's work. 
```
not necessarily bored - uneasy, anxious, malaise, etc.

Lior539 replied:
I think "unsatisfied" sums it up nicely

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+This has many benefits, like knowledge sharing across teams and diversifying skill sets. Most importantly, it significantly reduces the risk of losing a talented engineer.
+
+## Further reading
```
I think this would be a good place to call out alexeymk.com/growth-eng, with something like

> These days, Alexey lives in Tokyo and advises companies about Growth Engineering. If you're spinning up a new growth team or hoping to get your team to hit the high notes, [Alexey](alexeymk.com/growth-eng) can help.

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
```
one piece that's missing here is why these three categories are relevant.

Experiment Velocity (Engineer-weeks per experiment) * Experiment Win Rate * Average Experiment Win Size = your team's impact per engineer

IE, 2 eng-weeks per experiment * 40% win rate * 5% avg win size = 1% impact per eng-week.

So if you have 3 engineers and a quarter (12 weeks) you expect to move the metric by 36% that quarter 

But if you boost your win rate to 50%, it goes to 45%, or if get to 1.5 eng-weeks per experiment, the impact goes to  48%.

(Keep in mind that in practice, you'll want to roughly halve that estimate due to Winner's Curse https://medium.com/airbnb-engineering/selection-bias-in-online-experimentation-c3d67795cceb)

Lior539 replied:
Thanks! I want to jump straight into the post, so I think I'll just remove this section

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
```
I'm not sure if this is exactly the right hook. I think you are considering between two hooks here:

1. Sometimes a growth team is in a rut and here is how you fix it (in which case the first topic is more relevant than the last two)
2. Performance Tuning Growth Team - IE, the framework for thinking about improving your team's performance and ideas for improving each part.

I think 2 is more interesting philosophically, but I get why 1 might be a better hook - but then I would cut some of the non-engineering related parts in the name of keeping it tight. I rambled a bit on our call but that doesn't mean we have to include all of it 😄

Lior539 replied:
I've removed this section now

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
```
```suggestion
There are telltale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of code quality being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```
Code isn't the problem, standards are (I think)
```suggestion
Slow velocity is usually a result of quality standards being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```

Lior539 replied:
rephrased to over-engineering

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+### How to fix it
+
+First, sit with the engineering team and prioritize your technical debt. Figure out what is *actually* slowing you down (or likely to) and by how much. Once you have a ranked list, assign 10%-20% of engineering time to clean this up.
```
Is this something growth should do or the rest of the org? Sounds a bit big for growth to be responsible for.

ivanagas replied:
Going to require a lot of buy-in up the change to restructure what eng works on.

Lior539 replied:
In the context of this blog, it's the growth engineering team. I dropped the "growth" part to avoid overly repeating myself. 

I can add it/rephrase if you feel its still unclear?

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+First, sit with the engineering team and prioritize your technical debt. Figure out what is *actually* slowing you down (or likely to) and by how much. Once you have a ranked list, assign 10%-20% of engineering time to clean this up.
+
+Second (and more importantly), urge your team to take shortcuts. The role of a growth engineer is not necessarily to build features, but rather to validate whether a feature should exist. 
```
Short cuts seems too negative.
```suggestion
Second (and more importantly), urge your team to ship less polished code. The role of a growth engineer is not necessarily to build features, but rather to validate whether a feature should exist. 
```

Lior539 replied:
Hmm shortcuts really encompasses well what Im trying to say here. It's not just about shipping less polished code, it's about finding the quickest way to validating something.

I tried a few other variations, but didn't quite like them:

"urge your team to find the quickest paths possible..."
"urge your team to identify the shortest routes..."

AlexeyMK replied:
I think shortcuts is fine, but we can also use 

"encourage your team to be scrappy"

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+With this in mind, a good team exercise is to look at your last 20 experiments and identify which ones you could have validated faster. Thomas Owers, former growth lead at [Let's Do This](https://www.letsdothis.com/), has [good advice](/blog/how-to-start-a-growth-team#thomass-golden-rule-make-your-code-changes-as-small-as-possible) on this:
+
+"Instead of asking 'How long will a project take?', ask 'What can you do in only 1 week? 3 days? Half a day?'. Often, you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
```
```suggestion
> "Instead of asking 'How long will a project take?', ask 'What can you do in only 1 week? 3 days? Half a day?'. Often, you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
```

Lior539 replied:
Had a look at this, didnt quite like it as much with a callout. The flow felt a bit off

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
+2. Your experiment win rate.
+3. Your experiment win size.
```
I don't immediately know what you mean by these. 
- Success rate? Positive result rate?
- Impact size? Effect size?

Lior539 replied:
Think I'll just remove this section

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+You keep trying new ideas, but none of your experiments are working. 
+
+"You need better ideas," says Alexey, "and, counter-intuitively, the way to do this is to spend more time adding clear estimates to your current ideas."
```
Maybe mention cost-benefit here? This quote is a bit unclear.

Lior539 replied:
Hmm we jump into it in the next paragraph.

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+## Problem 2: Your experiment win rate is low
+
+You keep trying new ideas, but none of your experiments are working. 
```
```suggestion
You keep experimenting, but none of them are succeeding. 
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+Typically, teams prioritize their roadmaps by using frameworks such as [RICE](https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/) and assigning t-shirt size estimates ("S", "M", "L"). This doesn't go deep enough and leads to teams wasting time by building pointless things. 
+
+Spending only five minutes estimating something your team will spend weeks on doesn't make sense.
```
Estimating what? How long it will take? Its impact?

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+Once you have shortlisted the ideas you want to work on, the next step is to estimate the dollar impact of each one. Because you'll know **exactly** what to expect from your experiments, the result is a higher win rate.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a handy [sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
```
Can you give a little about the EVELYN framework here?

It also isn't immediately obvious how changes to Facebook Messenger have a dollar impact? I've never seen an ad there (although I know there is connections to the business). Maybe clarify what an active user is worth?

ivanagas replied:
Actually, maybe you should remove this paragraph because you repeat it down below.

Lior539 replied:
reworked

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+Enter EVELYN. Alexey knew they needed a dollar estimate of the impact.
+
+"Instead of building the entire backend, we created a simple [fake door test](https://posthog.com/tutorials/fake-door-test) on the frontend. Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was actually the existing one-size-fits-all pricing.
```
```suggestion
"Instead of building the entire backend, we created a simple [fake door test](/tutorials/fake-door-test) on the frontend. Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was actually the existing one-size-fits-all pricing.
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+"At MasterClass, we had one-size-fits-all pricing. We knew we would eventually need to add tiered pricing, but this would be a huge, months-long company effort. We struggled to prioritize it."
+
+Enter EVELYN. Alexey knew they needed a dollar estimate of the impact.
```
Maybe a graphic or list of what EVELYN stands for.

People should have all the information they need here, and not need to go to another page.

Lior539 replied:
reworked

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
```
Touch on this here: https://newsletter.posthog.com/i/136043854/you-need-a-culture-of-experimentation

Booking.com is a good example of this: https://hbr.org/2020/03/building-a-culture-of-experimentation

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Increase the number of high-quality project ideas.
+2. Maximize the impact of every experiment you ship.
```
This feels a bit weak. "When experiments have less impact, just maximize the impact of them."

Lior539 replied:
rephrased

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,133 @@
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Maximize the impact of every experiment you ship
```
Echoing my comment above, this feels weak. How do they improve the initial hypothesis? How do they know which ones will have an impact and which ones won't? If just shipping more iterations, that should be the advice.

Lior539 replied:
reworked

ivanagas reviewed (commented):
Some good stuff in here, but it feels like a lot of it sums up to "have more energy, have more ideas, ship more stuff" which isn't super novel. Would really appreciate one layer deeper in a bunch of places. 

A bit more depth in how, why, and examples could go a long way.

ivanagas commented:
I think some posts [Alexey's blog](https://alexeymk.com/) could provide the examples and depth this piece needs 😄

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
```
Did a second pass, seems to broadly make sense!

I think my biggest remaining issue is the negative slant "why your growth team isn't winning (and how to fix it)".

It's like if a personal trainer wrote a blog post saying "why you're fat."  ...sure, but now you're not the person I want to hear from now.

What's a more positive title we might take here?
- Helping your growth team hit the high notes?
- Growing your Growth team's impact?

etc

Lior539 replied:
Hmm interesting. I like this title as it sets up the rest of the post nicely i.e. the problems you need to dive into.

I'll give it some thought though and see if I can come up with something less negative

> It's like if a personal trainer wrote a blog post saying "why you're fat." ...sure, but now you're not the person I want to hear from now.

I think it's closer to "why you're not losing weight", which is not as negative :)

Lior539 replied:
How about something along the lines of: 
- The formula of a successful growth team

One concern would be that a title like that may mean changing the "problem - how to fix it" format of the blog  (and I quite like it!).

I'll update this PR with an update and see how it works

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+In fact, it's not uncommon for growth teams to cause a few incidents! And their goal must not be to avoid them.
+
+"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they resolve them quickly."
```
wait, the cool thing about this anecdote is that the crash rate didn't go down! Don't miss that part.

Lior539 replied:
Hmm not sure we need to include this to make the point 🙈

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of [over-engineering](https://www.joelonsoftware.com/2001/04/21/dont-let-architecture-astronauts-scare-you/). "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```
"Engineers hate doing growth" is a little extreme.   I'm an Engineer, I love it.

More... "Many engineers don't get along with a growth working style right away"

AlexeyMK commented:
> Some good stuff in here, but it feels like a lot of it sums up to "have more energy, have more ideas, ship more stuff" which isn't super novel. Would really appreciate one layer deeper in a bunch of places.
> 
> A bit more depth in how, why, and examples could go a long way.

Right - I think this summarizes what was bugging me as well, a lot of this is pretty superfluous.

I think one novel idea that I don't see that often is "Win rate * win size * velocity" formula, because it gives a nice way to both
1. estimate your team's impact during planning but also
2. identify what to focus on for fixing your team.

Adding to my own blogpost backlog :)

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+Then, use this dollar amount together with your effort estimates to calculate a prioritization score:
+
+![How to calculate your EVELYN score](../images/blog/how-to-fix-your-growth-team/evelyn-score.png)
```
Nice

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+![How to calculate your EVELYN score](../images/blog/how-to-fix-your-growth-team/evelyn-score.png)
+
+Because you'll have a clearer idea of what to expect from your experiments, the result is a higher win rate.
```
```suggestion
Because you'll have a clearer idea of what to expect from your experiments, you'll prioritize ones more likely to win.
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
```
```suggestion
When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVELYN framework mentioned above, you can prioritize and distill the list to only the best ideas.
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
```
Idk if this adds anything, especially if we are linking to another site.

Lior539 replied:
Yeah. Im 50/50 on it. The idea was to give an actionable tip on how to get your entire team to contribute ideas. I should probably just write that then. Will give it some thought

Lior539 replied:
Had a look into this. I dont think there's much more to add into how to create a bottoms-up idea culture that will be meaningful to add (I think it will be quite generic advice like "recognize peoples wins, create feedback channels etc".

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+- Engineers work to improve the initial hypothesis.
+- Engineers develop further iterations to test. 
+- Removing processes that prevent engineers from making quick decisions on their own.
+- Allowing engineers space to fail without being harsh on them.
```
```suggestion
- Giving engineers space to fail without punishment.
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+Because you'll have a clearer idea of what to expect from your experiments, the result is a higher win rate.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a [handy sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
```
I think it is worth noting that the calculation graphic above is the EVELYN framework. Without that, readers don't have any way to know what EVELYN is (without clicking out of the site).

Lior539 replied:
EVELYN is slightly deeper than this:

> . The idea is it’s pretty much a experiment tracker, an organizing tool that allows you to put in all your experiments, prioritize them well, track them from specing, designing, building and completion and also see how you did, like opportunity sizing on the way in, look at the results on the way out, and overall give you a set of processes and a storage source of truth for your experiments that I think can really help a team accelerate how they build and build the right experiments, getting you to more growth wins faster.

I dont really want to go to deep into EVELYN, as I think thats a blog post in itself. I think the important bit is the dollar amount tracking.

I might reach out to Darius at some point and see if we can write an EVELYN guide with him

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+In the context of growth teams, there are a few ways to do this:
+
+- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [configurators or feature flags](https://posthog.com/blog/feature-flags-vs-configuration).
+- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.
+- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.
+- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
```
```suggestion
- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [feature flags](/blog/feature-flags-vs-configuration).

- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.

- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.

- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+It's common for engineers to start feeling unsatisfied after a few quarters, and there's no shame in it. A symptom of this is diminishing returns from the team's work. 
+
+There's an easy fix for it, though:
```
```suggestion
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause a few incidents! And their goal must not be to avoid them.
```
```suggestion
In fact, it's common for growth teams to cause a few incidents! They won't succeed if they are focused on avoiding them.
```

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,141 @@
+
+Once you have shortlisted the ideas you want to work on, the next step is to **estimate the dollar impact of each one**.  
+
+Then, use this dollar amount together with your effort estimates to calculate a prioritization score:
```
Maybe mention that this is EVELYN here?

Lior539 replied:
mentioned below. The idea is to describe it first without naming it, so we jump straight into it

ivanagas reviewed (approved):
Looks good, some comments

Lior539 commented:
> I think one novel idea that I don't see that often is "Win rate * win size * velocity" formula, (https://github.com/PostHog/posthog.com/pull/6741#discussion_r1325963706) because it gives a nice way to both

Thanks for this feedback! I've added a section on "the growth team formula" now

AlexeyMK commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,155 @@
+
+Because you'll have a clearer idea of what to expect from your experiments, you'll prioritize the ones that are more likely to win.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a [handy sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
```
nit: I don't think he was VP-level at Facebook. https://www.linkedin.com/in/dariusmc/.

Either: 

- Head of Product Growth for Messenger
- VP Growth for Airtable

andyvan-ph commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,155 @@
+- Engineers work to improve the initial hypothesis.
+- Engineers develop further iterations to test. 
+- Removing processes that prevent engineers from making quick decisions on their own.
+- Giving engineers space to fail without being harsh on them.
```
It's jarring that you change the format of the article in this section – a format that really works btw.

Also, what constitutes a low win size? This is context readers need to make this useful.

andyvan-ph reviewed (commented):
High-level feedback only for now:

- This is shaping up nicely. I love the format for each section. Feels very clear and actionable.
- As per comment, I think we should revisit the headline, but that can wait.
- Main feedback is there just a few points where I'm left with too many questions, which is frustrating as a reader. Have flagged those where relevant.

Lior539 commented:
Latest changes:
- Changed the title to "5 problems every growth team faces (and how to fix them)", and tweaked the content accordingly
- Removed the references to EVELYN, since it felt like it was causing more confusion than clarity. Instead, I've added an example prioritization table directly in the post

Lior539 commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+---
```
I need to change the url path of this post still (since it has a new title). I'll do this just before shipping, since I dont want to lose any previous PR comments by renaming this file

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Make it everyone's job to come up with project ideas.
+2. Make your engineers own their experiments.
```
Potentially don't need these if you're going to use them as titles directly after?

ivanagas commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+
+In fact, it's common for growth teams to cause a few incidents! And they won't succeed if they are focused on avoiding them.
+
+"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they can resolve them quickly."
```
Mention Alexey again here, it's unclear where this quote is coming from.

ivanagas reviewed (approved):
Looks good, some final notes

andyvan-ph commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+  - Product engineers
+---
+
+There are telltale signs when a growth team is struggling: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/) has seen it all. He's worked in growth for more than 8 years. Most recently, he was [MasterClass's](https://www.masterclass.com/) head of growth engineering, where he grew the growth engineering org to 25 people across 5 teams.
+
+Alexey shared his advice with us on tackling the most common problems growth teams face.
```
```suggestion
Stuck metrics. Missed KPIs. Burnt out engineers. These are the telltale signs of a growth team in crisis. But how do you fix it?

If anyone knows, it's [Alexey Komissarouk](https://alexeymk.com/). He's worked in growth for eight years. First at Opendoor, followed by a three-year stint as Head of Growth Engineering at MasterClass, where he grew the growth engineering org to 25 people across 5 teams.

These are the most common growth problems he's encountered and how to fix them.
```

andyvan-ph reviewed (commented):
Left some edit suggestions, but this final polish is great. The whole section on dollar impact is so much clearer now.

I'm still kind of stuck on the headline. This one's definitely better, but I still think there's a better one.

We could lean into Alexey's credibility:

- Fixing common growth problems with ex-Head of Growth at MasterClass
- Growth failure modes (and how to fix them) with ex-Head of Growth at MasterClass

Or just a twist on the current one:

- The 5 most common growth failure modes (and how to fix them)

I know the piece is more about growth _teams_, but I think it's better to avoid that in the headline. It feels more broadly applicable and there 100% lessons for startups who don't have a formal growth team yet.

Lior539 commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+### How to fix it
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Make it everyone's job to come up with project ideas.
+2. Make your engineers own their experiments.
```
```suggestion
```

Lior539 commented on contents/blog/how-to-fix-your-growth-team.md:
```diff
@@ -0,0 +1,136 @@
+
+In fact, it's common for growth teams to cause a few incidents! And they won't succeed if they are focused on avoiding them.
+
+"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they can resolve them quickly."
```
```suggestion
"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they can resolve them quickly", says Alexey.
```
